about_atoms.exs

```diff
@@ -15,6 +15,15 @@ defmodule AboutAtoms do
     assert __? == list[:name]
   end
 
+  think "Atom keys enable a different access syntax" do
+    map = %{name: "Jay"}
+    assert map.name == __?
+
+    map = %{"name" => "Jay"}
+    attempt = fn -> __? end
+    assert_raise KeyError, "key :name not found in: %{\"name\" => \"Jay\"}", attempt
```

Comment:
Also we might illustrate that dot syntax fails for a missing key, where Access (subscript) syntax is more forgiving by returning `nil`.